src/preset_cli/cli/superset/sync/dbt/command.py

```diff
@@ -84,7 +84,7 @@ def dbt_core(  # pylint: disable=too-many-arguments, too-many-locals
     profiles: Optional[str] = None,
     exposures: Optional[str] = None,
     import_db: bool = False,
-    disallow_edits: bool = True,
+    disallow_edits: bool = False,
```

Comment:
I noticed this was set to `True` here even tough the default state is `False`, so I updated only for consistency but it doesn't really change the result. It should be totally fine to revert it to `True` if we want to.